apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -10,6 +10,34 @@
 
 namespace A8C\FSE\Coming_soon;
 
+$blog_locale = function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
+
+function get_redirect_to() {
+
+	// redirect_to is probably never set so this is almost pointless
```

Comment:
I see this is coppied from the existing v1 coming soon page. I feel like the tone of the comment is almost defeatist 😄😛.

Reply:
haha

I meant to remove that entirely :) Thanks for the reminder